lib/random.js

```diff
@@ -1,8 +1,20 @@
 'use strict';
 
+/**
+ * Return only one random item from an array.
+ *
+ * @since v0.0.1
+ * @param {Array} arr
+ * @return One random item.
```

Comment:
@erikjung Not that I can find. Some articles say the type is optional, others say it isn't. I haven't found anything concrete yet.